rollup.config.js

```
@@ -144,6 +144,41 @@ function config({ location, pkgJson, verboseMode }) {
+
+      if (compress) {
+        const glslifyPluginIdx = bundledPlugins.findIndex((item) => item === glslPlugin);
+        bundledPlugins.splice(
+          glslifyPluginIdx,
+          1,
+          glsl({
+            include: [/\.(glsl|gs)$/],
+            compress: true
+          })
+        );
+        bundledPlugins.push(minify({
```

ESLint

[error] Insert `⏎··········`

(prettier/prettier)